.github/ISSUE_TEMPLATE/2_bug_report_windows.yml

```diff
@@ -0,0 +1,106 @@
+name: "Report a Windows Issue"
+description: "Create a report for a runtime related Windows Issue"
+labels: ["os:win"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv version
+           - Windows Version
+           - Source of the mpv binary
+           - If known which version of mpv introduced the problem
+           - GPU model, driver and version
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
```

Comment:
For Windows and macOS we can mention or CI artifacts. Tell users to go to https://github.com/mpv-player/mpv/actions/workflows/build.yml?query=branch%3Amaster+event%3Apush and download latest build and test.